roles/reproducer/tasks/configure_computes.yml

```diff
@@ -19,6 +19,24 @@
         cmd: |
           ping -c2 {{ _controller_ip4 }}
 
+    - name: Tweak dnf configuration
```

Comment:
A couple of comments, nothing really blocker:

- The title of the block is not describing what's done with this change.
- The dnf tweak may not be "directly" related to the purpose of the change.

Reply:
It's part of it in fact - all related to "consume proxy". We're really applying DNF configuration, i.e. tweaking it - the CA is injected after it. Not really sure it deserves its own patch for the tweaking though...